### PR TITLE
feat(config): Disable timer feature

### PR DESCRIPTION
**Pull request by Scroody**
### What does this PR do?

This PR adds the timer feature to the list of disabled features.

### Closes Issue(s)

#18137

### CODE REVIEW

**Comment by antobinary** [comment]
Please also add this addition in api.md (see how it was mentioned in 2.6)
Screenshot from 2023-06-13 16-34-29

File: bigbluebutton-html5/imports/ui/services/features/index.js [hunk 1]
```diff
@@ -85,5 +85,5 @@ export function isReactionsEnabled() {
 }
 
 export function isTimerFeatureEnabled() {
-  return getDisabledFeatures().indexOf('timer') === -1;
+  return getDisabledFeatures().indexOf('timer') === -1 && Meteor.settings.public.timer;
```
Comment (GuiLeme) [inline bigbluebutton-html5/imports/ui/services/features/index.js:88]:
```suggestion
  return getDisabledFeatures().indexOf('timer') === -1 && Meteor.settings.public.timer.enabled;
```
Also, consider changing other places in the code where `Meteor.settings.public.timer` is being used to use this `isTimerFeatureEnabled` function.

**Review by GuiLeme — CHANGES_REQUESTED** [review]
I also think there might need some changes in `akka` to prevent any interactions of the timer from the back-end. I can do a PR to your branch.

**Review by GuiLeme — APPROVED** [review]
LGTM!

File: bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx [hunk 2]
```diff
@@ -309,21 +304,17 @@ class ActionsDropdown extends PureComponent {
...
+              screenshareHasEnded();
+              this.setCameraAsContentModalIsOpen(true);
+            },
       });
     }
 
     return actions;
   }
 
   makePresentationItems() {
```
[the remark this answers is not in the document]
Comment (GuiLeme) [reply bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx:316]: It's decided to not fix more lint errors in this PR

File: bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx [hunk 3]
```diff
@@ -332,57 +323,57 @@ class ActionsDropdown extends PureComponent {
...
   }
   setPropsToPassModal(value) {
-    this.setState({propsToPassModal: value});
+    this.setState({ propsToPassModal: value });
   }
-  setForceOpen(value){
-    this.setState({forceOpen: value});
+  setForceOpen(value) {
+    this.setState({ forceOpen: value });
   }
```
[the remark this answers is not in the document]
Comment (GuiLeme) [reply bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx]: > It's decided to not fix more lint errors in this PR

See comment above

**Comment by gustavotrott** [comment]
@GuiLeme , I think it's enough to validate `CreateTimerReqMsgHdlr` on the backend.
Not necessary to repeat the validation for every action!

File: docs/docs/development/api.md [hunk 4]
```diff
@@ -104,6 +104,8 @@ Updated in 2.6:
 
 Updated in 2.7:
 
+- **create** - **Added:**  Added `timer`, `cameraAsContent` and `snapshotOfCurrentSlide` as options for `disabledFeatures=`.
```
Comment (antobinary) [inline docs/docs/development/api.md:107]:
```suggestion
- **create** - **Added:**  Added `timer`, added `disabledFeatures` options`cameraAsContent` and `snapshotOfCurrentSlide`.
```

**Comment by gustavotrott** [comment]
Related https://github.com/bigbluebutton/bigbluebutton/issues/14203

**Pull request merged**